tests/core/pyspec/eth2spec/test/bellatrix/block_processing/test_process_execution_payload.py

```diff
@@ -30,6 +30,11 @@ def run_execution_payload_processing(spec, state, execution_payload, valid=True,
     # Before Deneb, only `body.execution_payload` matters. `BeaconBlockBody` is just a wrapper.
     body = spec.BeaconBlockBody(execution_payload=execution_payload)
 
+    # After (and including Deneb), the validity of a `state` with respect to an `execution_payload`
+    # is dependent on an update to `state.latest_block_header`. For this suite of tests to succeed,
+    # we have to emulate the update, so we do it here:
+    state.latest_block_header.parent_root = spec.hash_tree_root(state.latest_block_header)
```

Comment:
this is a bit weird but now essentially `process_execution_payload` requires `process_block_header` to run so that this assignment executes